migrate-about-us.rb

```diff
@@ -93,21 +98,111 @@
     description: 'Bivan Alzacky Harmanto adalah salah satu siswa yang berhasil mengikuti IMO 2012 dan IMO 2013 dan menyumbangkan medali perunggu untuk Indonesia. Saat ini Bivan sedang menempuh studi S1 jurusan Ilmu Komputer di Korea Advanced Institute of Science and Technology (KAIST), yang merupakan universitas nomor 1 di Korea untuk bidang teknik. Saat ini, Bivan merupakan salah satu anggota tim koreksi KTO Matematika. Selain gemar  mengotak-atik soal-soal Matematika (khususnya geometri) dan komputer, Bivan juga gemar mengecek Facebook tanpa alasan yang jelas (sekadar buka dan lihat-lihat saja), membagi ceritanya di blognya, dan bermain game di laptopnya. Oh iya satu hal lagi, board game sekarang menjadi salah satu kegemarannya, berkat salah satu temannya di KAIST yang terbilang sangat gila terhadap board game.',
     image: 'bivan.jpg',
   },
-];
+]
+
+def get_panitia_or_admin
+  users = User.all
```

Comment:
Can do this on the ActiveRecord layer?

Possibly something like
```ruby
User.joins(:roles).where('role.name' => ['panitia', 'admin'])
```

(the above script might not work, you might need to tweak it)

Doing this on the ActiveRecord layer is advantageous for various reasons. For one, you just execute one SQL query and let the database do what it's supposed to do: filtering through data.